daphne_worker_test/wrangler.toml

```diff
@@ -77,6 +77,7 @@ preview_id = "<ignored>" # TODO(cjpatton) Figure out how to pick this.
 # production. In particular, they will not be passed as environment variables
 # as they are here. See
 # https://developers.cloudflare.com/workers/wrangler/commands/#secret.
+DAP_ADMIN_BEARER_TOKEN = "administrator bearer token" # SECRET
```

Comment:
Do we need to set this for the leader too?

Reply:
This feature is optional: If not set, then tasks can't be configured this way.